docs/configuration/display.rst

```diff
@@ -0,0 +1,111 @@
+Display Options
+---------------
+The Navit display is highly customisable and consists of the following components
+
+* Graphics driver (appropriate engine for your system, to draw everything)
+* GUI (enables user interaction and map display)
+* OSD (shows widgets on map screen)
+
+Graphics Driver
+---------------
+
+Different technologies can be used, to let Navit draw it's visual components. Not all might be available at your specific system
+
+The current list of available graphics drivers:
+
+* **android**, for the Android port
+* **cocoa**, for the iPhone port
+* **gtk_drawing_area**,  usually most appropriate on Linux desktop systems
+* **sdl**, render inside an X window, or direct to the Linux framebuffer, with min dependencies on external libraries.
+* **qt5**, render using Qt5 library, either using QWidgets or QML. On top of any Qt5 supported display technology.
+* **win32** - useable with <tt>gtk</tt> or <tt>internal</tt> GUIs for Windows systems only.
+
+Experimental/less maintained drivers:
+
+* **qt_qpainter**, render inside X window or on top of Qt Palmtop Environment.
+* **opengl**, rendering via OpenGL <!--is it some 3D or accelerated stuff? What about OpenGL ES?-->
```

Comment:
The html comment at the end of the line seems strange, doesn't it?

Reply:
Yep, leftover from a wiki comment. Good catch!